pokete_data/npcs.py

```diff
@@ -276,6 +276,83 @@
         "x": 107,
         "y": 25
     },
+    "npc_17": {
+        "texts": [" < Welcome to Agrawos!", " < Check out the MowCow-Burger restaurant, they have the best beaf!", " < If you wan't a challange check out the Pokete-Arena, they have some quite heavy Poketes there!"],
+        "fn": None,
+        "map": "playmap_39",
+        "x": 102,
+        "y": 5
+    },
+    "npc_18": {
+        "texts": [" < This is the Pokete Arena of Agrawos, only go there if you're sure you can beat the challange.", " < They got some very heavy Poketes in there!"],
+        "fn": None,
+        "map": "playmap_39",
+        "x": 126,
+        "y": 36
+    },
+    "npc_19": {
+        "texts": [" < This is the temple of the Wheeto, they have the goood stuff in there."],
+        "fn": None,
+        "map": "playmap_39",
+        "x": 20,
+        "y": 65
+    },
+    "npc_20": {
+        "texts": [" < Hello yall, this town is soo famous for its' creaaazy plants.", " < Especially Wheeto, from which healing potions can be made.", " < Want one?"],
+        "fn": "playmap_39_npc_20",
+        "map": "playmap_39",
+        "x": 176,
+        "y": 56
+    },
+    "npc_21": {
+        "texts": [],
+        "fn": "playmap_42_npc_21",
+        "map": "playmap_42",
+        "x": 7,
+        "y": 3
+    },
+    "npc_22": {
+        "texts": [" < Oh man, if only I had a big and tasty MowCow-Burger...!"],
+        "fn": None,
+        "map": "playmap_42",
+        "x": 22,
+        "y": 4
+    },
+    "npc_23": {
+        "texts": [" < I'm priest of the Wheeto.", " < Those Poketes make you craaaaaazy!", " < They have some speeezial abilities!"],
+        "fn": "playmap_43_npc_23",
+        "map": "playmap_43",
+        "x": 7,
+        "y": 4
+    },
+    "npc_24": {
+        "texts": [" < I'm the mayor of Agrawos, the greatest city around.", " < Check out the MowCow-Burger restaurant, which offers the best MowCow-Burgers and take a look at the Pokete-Arena.", " < And as a special tip, take a look at the temple of the Wheeto."],
+        "fn": None,
+        "map": "playmap_45",
+        "x": 9,
+        "y": 3
+    },
+    "npc_25": {
+        "texts": [],
+        "fn": "playmap_39_npc_25",
+        "map": "playmap_39",
+        "x": 4,
+        "y": 33
+    },
+    "npc_26": {
+        "texts": [" < Hhhello young trainer", " < I've lived a loong live, bbut now it's time to go.", " < And I'm trying to scatter all my belongings all over the world.", " < Since you're a Pokete trainer, you may have use fpor this healing potion!"],
```

Comment:
```suggestion
        "texts": [" < Hhhello young trainer",
                  " < I've lived a loong live, bbut now it's time to go.",
                  " < And I'm trying to scatter all my belongings all over the world.",
                  " < Since you're a Pokete trainer, you may have use for this healing potion!"],
```

Maybe also make the stuttering more clear:

```suggestion
        "texts": [" < H-h-hello young trainer",
                  " < I've lived a loong live, b-but now it's time to go.",
                  " < And I'm trying to scatter all my belongings all over the world.",
                  " < Since you're a Pokete trainer, you may have use for this healing potion!"],
```

Reply:
That's a good idea.